src/app/charts/core/files/secrets/tpl/spring-config-saml.properties

```diff
@@ -0,0 +1,45 @@
+{{- $samlconfig := (include "arkcase.saml" $) -}}
```

Comment:
I recommend using `fromYaml` here right away, so from here on in you'll be dealing with the actual configuration _object_ vs. the YAML string which is just mostly useless. this will keep you from having to parse the YAML over and over and over again later on, and just access your configuration values directly.